files/brews/git.rb

```diff
@@ -1,27 +1,26 @@
 class Git < Formula
   homepage "http://git-scm.com"
-  url "https://www.kernel.org/pub/software/scm/git/git-2.2.2.tar.gz"
-  sha256 "e9e50d0d382183a9327d39334eb65db92852208e1438aab2a9bf8d06ef17b653"
+  url "https://www.kernel.org/pub/software/scm/git/git-2.3.0.tar.gz"
+  sha256 "ba2fe814e709a5d0f034ebe82083fce7feed0899b3a8c8b3adf1c5a85d1ce9ac"
 
   head "https://github.com/git/git.git", :shallow => false
 
-  version "2.2.2-boxen1"
-
+  version "2.3.0-boxen1"
 
   bottle do
-    sha1 "ea73eb22e16fffb1a1a4f610c263d1883353a879" => :yosemite
-    sha1 "e74c6eb7ea70e88a60448c46c0956be17006d8cd" => :mavericks
-    sha1 "b422facacb49cf37a5488634324decfc4fc8440a" => :mountain_lion
+    sha1 "53177456ab367faa30e1d49104e8e2c065ac5613" => :yosemite
+    sha1 "549b0d4155ad0ca02edb0a55599006da32b21e00" => :mavericks
+    sha1 "78f9b2bfcdfe7e7d356cc4020df4ec41217f6e25" => :mountain_lion
```

Comment:
You should remove the `bottle do` block here because this bottle doesn't exist on Homebrew's servers with the version `2.3.0-boxen1`. Alternatively, just remove the `version` and use the Homebrew formula as-is so people with Boxen installed in `/usr/local` can use it.